### PR TITLE
[#55009] 14.1 Documentation: Update permissions  https://community.op…

### PR DESCRIPTION
**Pull request by MayaBerd**
…enproject.org/work_packages/55009

[#55009] 14.1 Documentation: Update permissions

https://community.openproject.org/work_packages/55009

### CODE REVIEW

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Following are the permissions for file storages within OpenProject:
...
 Following user permissions are set on files and folder in **External Storages**:
 
-- **External Storage: Read files (Nextcloud, OneDrive/SharePoint)**
-- **External Storage: Write files (Nextcloud, OneDrive/SharePoint)**
-- **External Storage: Create files (Nextcloud)**
-- **External Storage: Share files (Nextcloud)**
-- **External Storage: Delete files (Nextcloud)**
+- **Automatically managed folders: Read files (Nextcloud, OneDrive/SharePoint)**
+- **Automatically managed folders:  Write files (Nextcloud, OneDrive/SharePoint)**
+- **Automatically managed folders: : Create files (Nextcloud)**
```
Comment (dominic-braeunlein) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]:
```suggestion
- **Automatically managed folders: Create files (Nextcloud)**
```

Comment (dominic-braeunlein) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]:
```suggestion
- **Automatically managed folders: Create files (Nextcloud)**
```

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ Following are the permissions for file storages within OpenProject:
...
+- **Automatically managed folders:  Delete files (Nextcloud)**
```
Comment (dominic-braeunlein) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]:
```suggestion
- **Automatically managed folders: Delete files (Nextcloud)**
```

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 3]
```diff
@@ -29,11 +29,11 @@ Following are the permissions for file storages within OpenProject:
...
+- **Automatically managed folders:  Share files (Nextcloud)**
```
Comment (dominic-braeunlein) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]:
```suggestion
- **Automatically managed folders: Share files (Nextcloud)**
```

**Review by dominic-braeunlein — CHANGES_REQUESTED** [review]
Looks good!
I saw some small formatting errors

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 4]
```diff
@@ -17,23 +17,23 @@ Permissions control what users can see and do within OpenProject. Permission are
...
 
-- **External Storage: Read files (Nextcloud, OneDrive/SharePoint)**
-- **External Storage: Write files (Nextcloud, OneDrive/SharePoint)**
-- **External Storage: Create files (Nextcloud)**
-- **External Storage: Share files (Nextcloud)**
-- **External Storage: Delete files (Nextcloud)**
+- **Automatically managed folders: Read files (Nextcloud, OneDrive/SharePoint)**
+- **Automatically managed folders:  Write files (Nextcloud, OneDrive/SharePoint)**
+- **Automatically managed folders: Create files (Nextcloud)**
+- **Automatically managed folders:  Delete files (Nextcloud)**
```
Comment (MayaBerd) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]:
```suggestion
- **Automatically managed folders: Delete files (Nextcloud)**
```

**Pull request merged**